core/src/identity/error.rs

```diff
@@ -67,13 +67,15 @@ pub struct SigningError {
 
 /// An error during encoding of key material.
 impl SigningError {
+    #[allow(dead_code)]
```

Comment:
When does this trigger? Do we only use this function with the `rsa` feature?

Reply:
SigningError used in `rsa` and `secp256k1` features. So with these features disabled, these functions become dead code. Probably, `#[cfg(any(feature = "secp256k1", feature = "rsa"))]` would be better here.

Reply:
I think so too, can you change it to the above feature combination?

Reply:
Ok, replaced `allow(dead_code)` with `cfg(feature=...)`